packages/adaptive_scaffold/pubspec.yaml

```diff
@@ -8,14 +8,14 @@ publish_to: none
 
 environment:
   sdk: '>=2.17.0 <3.0.0'
-  flutter: ">=3.0.0"
+  flutter: ">=1.17.0"
```

Comment:
Flutter version should be what I had from before since that's what the packages support.